plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/glue/GlueMetastoreModule.java

```diff
@@ -126,6 +126,7 @@ public static GlueClient createGlueClient(GlueHiveMetastoreConfig config, OpenTe
         GlueClientBuilder glue = GlueClient.builder();
 
         glue.overrideConfiguration(builder -> builder
+                .addExecutionInterceptor(new GlueHiveExecutionInterceptor(config))
```

Comment:
isnt it better to add the interceptors (line 130) involving telemetry earlier in the chain?

Reply:
I don't know